package.json

```diff
@@ -47,6 +48,7 @@
     "express-naked-redirect": "^0.1.2",
     "express-session": "^1.16.2",
     "express-static-gzip": "^0.2.2",
+    "feed": "^4.2.2",
```

Comment:
_non-blocking_

I wondered if `feed` should be under `devDependencies` because the feed is only updated during build time, but realized that's not possible with the way this has been implemented. If `scripts/generate-blog-feeds.js` were the only thing that used `blogPosts.js` then it should be a dev dependency. However, Next.js uses `getBlogPosts()` (in the same file) during runtime.

If this is reorganized so that `generateBlogFeeds()` is not in the same file as `getBlogPosts()`, `feed` could be made a dev dependency. I'll let you decide if that's worthwhile.